test/fixtures/templater/jinja_l_metas/001.yml

```diff
@@ -28,16 +28,18 @@ file:
             - dedent: ''
             - from_clause:
               - keyword: FROM
+              - indent: ''
               - whitespace: ' '
               - from_expression:
-                  indent: ''
                   from_expression_element:
                     table_expression:
                       table_reference:
                       - identifier: DATAHUB
                       - dot: .
                       - identifier: SNAPSHOT_DAILY
                   dedent: ''
+                  indent: ''
```

Comment:
I don't understand why we have these two? A `dedent` immediately followed by an `indent`? Is this because of the first change above on 1271/1282?